lib/services/api/user_token_service.rb

```diff
@@ -32,9 +32,9 @@ def generate_token(userid, requester_type)
 
       # Additional Requester type token ttl's for authentication
       type_to_ttl_override = {'ui' => ::Settings.session.timeout}
-
-      token_mgr(requester_type).gen_token(:userid             => userid,
-                                          :token_ttl_override => type_to_ttl_override[requester_type])
+      token_metadata = { :userid => userid, :token_ttl_override => type_to_ttl_override[requester_type] }
```

Comment:
@abellotti This is where the only conflict was. I made the same change (passing `type_to_ttl_override[requester_type]`) as we did on the appliance.

Reply:
yep makes sense, gotta get the right expected metadata for this release.